pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Why did you make this change?
+(cite Issue # OR provide rationalization of changes if no issue exists)
+
+
+
+## What did you do? 
+- [add implementation details here]
+- 
+
+
+## How to test the changes? 
+(select the most appropriate option; if the latter, provide steps for testing below)
+- [ ] I've included a unit or integration test
```

Comment:
```
- [ ] I've included an appropriate [automated test](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
```

or

```
- [ ] I've included an appropriate automated test.
```

Reply:
or plural - there may be multiple tests.

Reply:
I liked the idea of having a link to https://docs.galaxyproject.org/en/latest/dev/writing_tests.html to help first-time contributors, was it dismissed?

Reply:
It wasn't dismissed - I just took the low hanging fruit option. I can add that in with my next commit.